src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double sideLength;
+
+    public Square(Color color, double sideLength) {

```

Comment:
Passing expressions directly in the constructor can reduce code readability. Consider creating a variable for 'sideLength' before passing it to the constructor if this value is derived from complex expressions.